rust/abacus-base/src/contract_sync/outbox.rs

```diff
@@ -260,160 +307,100 @@ mod test {
             let latest_valid_message_range_start_block = 100;
 
             let mut mock_indexer = MockAbacusIndexer::new();
+            let mut mock_cursor = MockSyncBlockRangeCursor::new();
             {
                 let mut seq = Sequence::new();
 
+                // Some local macros to reduce code-duplication.
+                macro_rules! expect_current_position {
```

Comment:
nice idea putting these into macros

Reply:
Yeah, I was getting tired of both how verbose the code was getting and also how unreadable it was.